vendor/engines/bulk_email/app/controllers/bulk_email/bulk_email_controller.rb

```diff
@@ -21,15 +21,12 @@ class BulkEmailController < ApplicationController
     def search
       @searcher = BulkEmailSearcher.new(@search_fields)
       @users = @searcher.do_search
+    end
 
-      respond_to do |format|
-        format.html
-
-        format.csv do
-          filename = "bulk_email_#{params[:bulk_email][:user_types].join('-')}.csv"
-          set_csv_headers(filename)
-        end
-      end
+    def download
+      @users = User.where_ids_in(params[:recipient_ids])
```

Comment:
I don't think we need to worry about this, but should we add any sort of constraints around this? People could get access to other facilities user lists. Maybe `BulkEmailSearcher` has an `all` method that gives us the widest list within a facility. Not sure if it's worth dealing with, but something to think about.